src/MockClassifier.Api/Services/NaturalLanguageService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MockClassifier.Api.Models;
+using MockClassifier.Api.Interfaces;
+
+namespace MockClassifier.Api.Services
+{
+    public class NaturalLanguageService : IClassifierService
+    {
+
+        public string[] Classify(string messageBody)
+        {
+            var token = "";
+            switch (messageBody)
+            {
+                case "I want to register my child at school": token = Ministry.education.ToString(); break;
+                case "How do I file my annual tax information": token = Ministry.economic.ToString(); break;
+                case "I have a question about the Estonian pension system": token = Ministry.economic.ToString(); break;
+                case "How do I get to Lahemaa park?": token = Ministry.environment.ToString(); break;
+                case "How do I arrange for my COVID-19 booster vaccination": token = Ministry.social.ToString(); break;
+                case "I wish to understand what benefits my family are entitled to": token = Ministry.social.ToString(); break;
+            }
+
+            return new string[] { token };
```

Comment:
I think this functionality should move into `TokenService` class. We would need this as a fallback in case there are no tokens in the message.

Also, it would save us from injecting multiple `IClassifierService` services into the same controller.

Reply:
@decodingahmed we've deliberately separated this out into two distinct services; one that deals with tokens and one that deals with natural language. The intention here is on the basis that the natural language bit may grow over time (Cognitive Language Service) even in the context of the mock classifier. Having them separate makes more sense regarding separation of concerns etc. Regarding fall back, if there are no tokens, that is a perfectly normal path and it just means that the token service does not return any tokens (which is fine), we do not need a fall back there.